core/Property/Property.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Property;
+
+use Exception;
+use Piwik\Site;
+
+/**
+ * Provides access to individual properties.
+ */
+class Property extends Site
```

Comment:
We will most likely change it later. For now not really important. I only want to add this feature on top of the current solution as simple as possible so we can later easily change it again